### PR TITLE
Remove warnings about experimental features, get rid of json syntax e…

### PR DESCRIPTION
**Pull request by jobergum**
@vekterli  could you please have a look at the timestamp/ordering stuff and add commits to this branch. Thanks

### CODE REVIEW

File: documentation/writing-to-vespa.html [hunk 1]
```diff
@@ -186,43 +181,6 @@ <h2 id="capacity-and-feed">Capacity and feed</h2>
...
-Put operation rejected for document 'id:test:test::0' of type 'test': 'multiValueLimitReached: {
-  action: \"use 'huge' setting on attribute field or add more content nodes\",
-  reason: \"multiValue address space used (0.55) &gt; limit (0.5)\",
-  multiValue: {
-    used:  73819750,
-    limit: 134217728
-  },
-  attributeName: \"a1\",
-  subdb: \"ready\"
-}'
```
Comment (kkraune) [inline documentation/writing-to-vespa.html]: good, it suffices with one example :-)

File: documentation/writing-to-vespa.html [hunk 2]
```diff
@@ -248,166 +206,3 @@ <h2 id="performance">Performance</h2>
...
-    &lt;latency average="0.25708" last="0.25708" min="0.25708" max="0.25708" count="1"/&gt;
-  &lt;/foo&gt;
-  &lt;sum&gt;
-    &lt;count count="2"/&gt;
-    &lt;latency average="0.191528" last="0.25708" min="0.125977" max="0.25708" count="2"/&gt;
-    &lt;not_found count="1"/&gt;
-  &lt;/sum&gt;
-&lt;/get&gt;
-</pre>
-</p>
```
Comment (kkraune) [inline documentation/writing-to-vespa.html]: I agree the section is not good. but we lack a description of this feature in vespa, albeit in another form or elsewhere. please keep it for now and I can rewrite it

File: documentation/writing-to-vespa.html [hunk 3]
```diff
@@ -248,166 +206,3 @@ <h2 id="performance">Performance</h2>
...
-<table class="table">
-  <thead>
-  <tr><th>Partial update contains</th><th>Result</th></tr>
-  </thead></tbody>
-  <tr><td>field1</td><td>field1 is updated</td></tr>
-  <tr><td>field2</td><td>The update fails</td></tr>
-  <tr><td>field1 and field2</td><td>field1 and field2 is updated</td></tr>
-  </tbody>
-</table>
-</p>
```
Comment (kkraune) [inline documentation/writing-to-vespa.html]: This section should be moved to search-definitions.html - i will add it there, so ok to remove from here

File: documentation/writing-to-vespa.html [hunk 4]
```diff
@@ -248,166 +206,3 @@ <h2 id="performance">Performance</h2>
...
-        }
-    }
-}
-</pre>
-        Feed the update feed using the <a href="vespa-http-client.html">Vespa HTTP client</a>.
-        Verify that the update is visible and searchable by searching for <em>tags:michael</em>.
-      </td>
-    </tr>
-  </tbody>
-</table>
```
Comment (kkraune) [inline documentation/writing-to-vespa.html]: OK to remove these examples, we have examples elsewhere

**Comment by kkraune** [comment]
I am good with merging this, thanks for cleaning up

File: documentation/writing-to-vespa.html (hunk 2 above)
Comment (jobergum) [reply documentation/writing-to-vespa.html]: We don't need this. This is reference material.

**Pull request merged**